lib/walex/event/dsl.ex

```diff
@@ -1,6 +1,7 @@
 defmodule WalEx.Event.Dsl do
   defmacro process_events_async(events, functions) do
-    module = hd(__CALLER__.context_modules)
+    [_ | _] = __CALLER__.context_modules
```

Comment:
Is this line useful?

Reply:
yes, context_modules can be an empty list according to its type
if so this macro should crash